README.md

```diff
@@ -1,23 +1,23 @@
-# Overlay-Specification
+# Overlay Specification
 
-The Overlay Specification is a community-driven open specification within the [OpenAPI Initiative](https://www.openapis.org/), a Linux Foundation Collaborative Project.
+The Overlay Specification defines a document format for information that will augment an existing OpenAPI description without entangling that information in the OpenAPI description's source doccument(s).
 
-The Overlay specification defines a way of creating documents that contain information to be merged with an OpenAPI description at some later point in time, for the purpose of updating the OpenAPI description with additional information.
+This specification is a community-driven, open specification within the [OpenAPI Initiative](https://www.openapis.org/), a Linux Foundation Collaborative Project.
 
-Overlays can address a wide range of scenarios that have been frequently requested by OpenAPI users:
+Overlays support a range of scenarios, including:
 
-- Support multi-language API descriptions by using Overlays to contain language translations.
-- Provide configuration information for different deployment environments.
-- Allow separation of concerns for metadata such as gateway configuration or SLA information.
-- Support a traits like capability for applying a set of configuration data, such as multiple parameters, or multiple headers to a targeted object.
-- Provide default responses, or parameters where they are not explicitly provided.
-- Apply configuration data globally or based on filter conditions.
+- Translating documentation into another language
+- Providing configuration information for different deployment environments
+- Allowing separation of concerns for metadata such as gateway configuration or SLA information
+- Supporting a traits-like capability for applying a set of configuration data, such as multiple parameters or multiple headers, for targeted objects
+- Providing default responses or parameters where they were not explicitly provided
+- Appling configuration data globally or based on filter conditions
 
 ## Current Status
 
-The current specification is sufficiently stable for implementers to start experimenting.  We are looking for implementation experience to guide our decisions on the remaining open issues.
+The current specification is sufficiently stable for implementers to start experimenting. We are looking for implementation experience to guide our decisions on the remaining open issues.
 
-## Tools that support Overlays
+## Tools that Support Overlays
```

Comment:
Let's move implementations into a separate file that is not part of the specification.  We can create a link to that file.  Having to release a new patch of the spec to add an implementation seems like it would be painful.